examples/eks-cluster-with-self-managed-node-groups/main.tf

```diff
@@ -123,18 +123,22 @@ module "aws-eks-accelerator-for-terraform" {
 
       block_device_mappings = [
         {
+          # since kms_key_id is not specified uses default aws/ebs AWS managed key
           device_name = "/dev/xvda" # mount point to /
           volume_type = "gp2"
           volume_size = 50
         },
         {
+          # since kms_key_id is specified uses the one declared
           device_name = "/dev/xvdf" # mount point to /local1 (it could be local2, depending upon the disks are attached during boot)
           volume_type = "gp3"
           volume_size = 80
           iops        = 3000
           throughput  = 125
+          kms_key_id  = "arn:aws:kms:us-west-2:123456789012:key/a4688694-fedc-2222-baee-11d4444b5c6f" #encrypt with specified kms key
```

Comment:
Agree with Apoorava.
@ricardosouzamorais I think its better you create a KMS key resource in the example using our own KMS module and pass the key as an input to this `kms_key_id`

Reply:
I think it's better to have this as optional because if the customer has default encryption enabled already in the region and account then this will override that key. We need to create a boolean var like "encrypt_volumes_custom_kms_key". If not, then we need to document it clearly.

Reply:
It was created this way now, @wsarwari-amzn .